pandas/tests/io/test_html.py

```diff
@@ -1233,3 +1234,15 @@ def run(self):
         while helper_thread1.is_alive() or helper_thread2.is_alive():
             pass
         assert None is helper_thread1.err is helper_thread2.err
+
+    def test_parse_Path_object(self):
+        """
+        read_html should be able to cope with Path
+
+        GH 37705
+        """
+        file_path_string = r"pandas/tests/io/data/html/spam.html"
```

Comment:
Rather than hard coding this can you use the `datapath` fixture? You will see it elsewhere in the test module

Reply:
Fixed accordingly. Thanks for the comments!